### PR TITLE
Add compat data for DocumentFragment

### PR DESCRIPTION
**Pull request by frankyjuang**
https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment

### CODE REVIEW

File: api/DocumentFragment.json [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentFragment",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "1"
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/DocumentFragment.json]: Should be `true`.

File: api/DocumentFragment.json [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentFragment/DocumentFragment",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "28"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DocumentFragment.json]: Both Android flavors should be `true`.

File: api/DocumentFragment.json [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+              "version_added": "24"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DocumentFragment.json]: Should be `true`.

File: api/DocumentFragment.json [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "querySelector": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentFragment/querySelector",
+          "support": {
+            "webview_android": {
+              "version_added": "2.1"
```
Comment (jpmedley) [inline api/DocumentFragment.json]: Change to `true`. I don't have the actual version, and webview follows Chrome version numbers, not Android.

File: api/DocumentFragment.json [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DocumentFragment.json]: Change to `true`.

File: api/DocumentFragment.json [hunk 6]
```diff
@@ -0,0 +1,295 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "querySelectorAll": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentFragment/querySelectorAll",
+          "support": {
+            "webview_android": {
+              "version_added": "2.1"
```
Comment (jpmedley) [inline api/DocumentFragment.json]: `true`.

File: api/DocumentFragment.json [hunk 7]
```diff
@@ -0,0 +1,295 @@
...
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DocumentFragment.json]: `true`.

File: api/DocumentFragment.json [hunk 8]
```diff
@@ -0,0 +1,295 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ParentNode#Properties",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "28"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/DocumentFragment.json]: `true`

**Comment by teoli2003** [comment]
@frankyjuang To help you fix the error, there has not been a Firefox 1 on Android. The oldest firefox_android version you can use is 4. Change the few "1" you have for Firefox_android to "4" and the errors will go away.

**Pull request merged**